tests/integration/security/testdata/requestauthn/f-authn.yaml.tmpl

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: "security.istio.io/v1beta1"
+kind: "RequestAuthentication"
+metadata:
+  name: "request-authn-for-f"
+  namespace: {{ .Namespace }}
+spec:
+  selector:
+    matchLabels:
+      app: f
+  jwtRules:
+  - issuer: "test-issuer-1@istio.io"
+    jwks: '{ "keys":[ {"e":"AQAB","kid":"abc","kty":"RSA","n":"failedToFetchJwtPubKey"}]}'
```

Comment:
Use example jwks URI here to make sure it is the same as if it's generated from the code, e.g. `"failedToFetchJwksForhttps://raw.githubusercontent.com/istio/istio/release-1.6/security/tools/jwt/samples/jwks.json"`